src/components/users/ForgotPwd.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useEffect } from 'react';
+import { func, object } from 'prop-types';
+import Loader from 'react-loader-spinner';
+import { useDispatch, useSelector } from 'react-redux';
+
+import '../../style/App.scss';
+import './user-form.scss';
+import FormInput from '../common/FormInput';
+import { userRequest, userActions } from '../../actions/user.actions';
+import { userConstants } from '../../constants/user.constants';
+
+const ForgotPwd = ({ intl, setForgotPwd }) => {
```

Comment:
you can use `useIntl` instead of used as a prop

Reply:
Ok I can change that in another PR to keep this one little